tests/e2e/release/cht-config/contact-edit.specs.js

```diff
@@ -58,4 +59,50 @@ describe('Editing contacts with the CHT config', () => {
     await contactPage.selectLHSRowByText(healtchCenterName);
     expect(await contactPage.peopleRows.count()).toBe(0);
   });
+
+  it('should change primary contact', async () => {
+    const contacts = [
+      {
+        _id: 'three_district',
+        type: 'district_hospital',
+        name: 'Maria\'s district',
+        contact: { _id: 'one_person' },
+        reported_at: new Date().getTime(),
+      },
+      {
+        _id: 'three_person',
+        type: 'person',
+        name: 'Maria',
+        parent: { _id: 'three_district' },
+        reported_at: new Date().getTime(),
+      },
+
+      {
+        _id: 'four_person',
+        type: 'person',
+        name: 'Marta',
+        parent: { _id: 'three_district' },
+        reported_at: new Date().getTime(),
+      },
+    ];
+
+    await utils.saveDocs(contacts);
+
+    await commonElements.goToPeople();
+    await contactPage.selectLHSRowByText('Maria\'s district');
```

Comment:
Once converted to factory objects, as mentioned above, can you change this from the string to use `contact.name`